### PR TITLE
Fixing error when using graph command and the configured remote is empty

### PR DESCRIPTION
**Pull request by talessatiro**
When the user types the command "ml-git repository graph" with a configured empty remote (there is no default branch created) the following error is shown:

image

This PR skip the problem to show only the message:

image

Depends on: [PR 143](https://github.com/HPInc/ml-git/pull/143)

### CODE REVIEW

File: ml_git/relationship/local_entity_manager.py [hunk 1]
```diff
@@ -52,17 +52,20 @@ def get_entities(self):
...
+                    continue
+                repository = metadata_repository(False, '', '', '', metadata_owner('', ''))
+                for obj in Repo(self._manager.path).head.commit.tree.traverse():
+                    if SPEC_EXTENSION in obj.name:
+                        entity_spec = yaml_load_str(io.BytesIO(obj.data_stream.read()))
+                        entity = Entity(repository, entity_spec)
+                        if entity.type in type_entity.value and entity not in entities:
+                            entities.append(entity)
+        except Exception as error:
+            log.debug(output_messages['DEBUG_ENTITIES_RELATIONSHIP'].format(error), class_name=LocalEntityManager.__name__)
```
Comment (gilMars) [inline ml_git/relationship/local_entity_manager.py:68]: We have test for this case?

Comment (talessatiro) [reply ml_git/relationship/local_entity_manager.py:68]: No, i'm gonna do that

Comment (talessatiro) [reply ml_git/relationship/local_entity_manager.py:68]: It was generated an API test to validate this scenario. But to validate successfully the test, it needs of the fix implemented in [PR 143](https://github.com/HPInc/ml-git/pull/143)

Comment (talessatiro) [reply ml_git/relationship/local_entity_manager.py:69]: The test was implemented and the branch was merged with the bugfix/2.2.2 [PR 143](https://github.com/HPInc/ml-git/pull/143)

**Review by gilMars — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**